test/cppsuite/test_harness/workload/database_operation.cxx

```diff
@@ -287,7 +287,7 @@ database_operation::update_operation(thread_context *tc)
 
         /* Commit the current transaction if we're able to. */
         if (tc->transaction.can_commit())
-            tc->transaction.commit();
+            WT_IGNORE_RET(tc->transaction.commit());
```

Comment:
Why are we ignoring the return value? Do we want the program to proceed if a failure occurs during commit?

Reply:
Yep! so `can_commit()` should cover this case. can_commit() can catch the failure. I wonder if we want to be extra safe and assert that this must be true? This could uncover some potential test framework bugs @etienneptl  what do you think?

Reply:
Since it is in `update_operation` of the base class, I would leave it this way. When one defines a very specific test, they probably want to be sure they know what is happening and they would assert the result, here, we are just providing an example.